app/assets/javascripts/upvote.js

```diff
@@ -0,0 +1,20 @@
+$(".vote").click(function()
+{
+
+  var voteType = this.id
+  var parentId = this.parentElement
+  var reviewId = parentId.parentElement.id
+  var url = window.location.pathname;
+  var foodTruckId = url.substring(url.lastIndexOf('/') + 1);
+  $.ajax({
+  METHOD: "POST",
+  url: "/food_trucks/" + foodTruckId + "/votes",
+  dataType: "JSON",
+  data: { _method:"put", review_id: reviewId, vote_type: voteType, food_truck_id: foodTruckId },
```

Comment:
Line is too long.<br>Identifier 'review_id' is not in camel case.<br>Identifier 'vote_type' is not in camel case.<br>Identifier 'food_truck_id' is not in camel case.